isb_web/main.py

```diff
@@ -210,6 +211,30 @@ async def login(request: fastapi.Request):
     return await _cli.authorize_redirect(request, OAUTH_REDIRECT_URL, state=state)
 
 
+@app.get("/orcid_token", include_in_schema=False)
+def get_orcid_token(request: fastapi.Request, response: fastapi.Response):
```

Comment:
@datadavev I saw that the GitHub OAuth implementation had some referrer checks.  Not sure if we need to add those here?  If so, presumably the only allowed host should be orcid?

Reply:
Technically (though very unlikely) it could be used by anyone as a callback for github authentication. The referer checks are there to reduce the likelihood of abuse of the service.